helm_deploy/apply-for-legal-aid/templates/_envs.tpl

```diff
@@ -185,6 +185,23 @@ env:
       secretKeyRef:
         name: {{ template "apply-for-legal-aid.fullname" . }}
         key: laaPortalMockSaml
+  - name: OMNIAUTH_AZURE_CLIENT_ID
+    valueFrom:
+      secretKeyRef:
+        name: laa-apply-for-legalaid-secrets
+        key: omniauthAzureClientID
+  - name: OMNIAUTH_AZURE_CLIENT_SECRET
+    valueFrom:
+      secretKeyRef:
+        name: laa-apply-for-legalaid-secrets
+        key: omniauthAzureClientSecret
+  - name: OMNIAUTH_AZURE_TENANT_ID
+    valueFrom:
+      secretKeyRef:
+        name: laa-apply-for-legalaid-secrets
+        key: omniauthAzureTenantID
+  - name: OMNIAUTH_AZURE_TENANT_ID
```

Comment:
Is this a typo? Should this be `OMNIAUTH_AZURE_REDIRECT_URI` instead?

Reply:
oops, yes it is. 👍